### PR TITLE
#65273520: Setting Travis CI

### PR DESCRIPTION
**Pull request by ZackAtama**
#### What does this PR do?

- Add Travis continuous integration to run test builds

#### Description of Task to be completed?

- Setup Travis CI configurations by creating the `.travis.yml` file with the setting to run the application builds

#### How should this be manually tested?

- It is an online build triggered by the `.travis.yml` file.

#### Any background context you want to provide?

- Tests must be passing in order for the builds to pass

#### What are the relevant pivotal tracker stories?

[#165273520](https://www.pivotaltracker.com/story/show/165273520)

##### Travis Badge in ReadMe
Screenshot 2019-05-29 at 14 48 03

##### Build in TravisCI
Screenshot 2019-05-29 at 14 41 11

### CODE REVIEW

**Review by nadralia — APPROVED** [review]
Nice Work @AtamaZack

File: ReadMe.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/AtamaZack/ah-frontend-dojo.svg?branch=chore-travisCI-setup-165273520)](https://travis-ci.com/AtamaZack/ah-frontend-dojo)
+
```
Comment (bisonlou) [inline ReadMe.md:2]: Good Work @AtamaZack. Though I wonder whether we will be using the badge attached to your repo or the Andela repo.

**Review by bisonlou — CHANGES_REQUESTED** [review]
Awesome work @AtamaZack. Kindly look and see if the changes I have requested can be made.

File: ReadMe.md (hunk 1 above)
Comment (kalsmic) [reply ReadMe.md:2]: Nice work @AtamaZack but I also think you should look into changing the reference of the badge to point to the Andela repo.

**Review by kalsmic — CHANGES_REQUESTED** [review]
@AtamaZack, kindly look into this feedback.

File: ReadMe.md (hunk 1 above)
Comment (ZackAtama) [reply ReadMe.md:2]: Thanks for the feedback, resolved.

**Review by kalsmic — APPROVED** [review]

**Review by bisonlou — APPROVED** [review]

**Review by llwasampijja — APPROVED** [review]
@AtamaZack nice work. Travis ci is well implemented and now the badges point to the correct repo!

**Review by JamesMudidi — APPROVED** [review]
Nice work zack. I approve

**Pull request merged**